api/src/opentrons/protocol_api/labware.py

```diff
@@ -883,7 +884,10 @@ def tip_length(self, length: float) -> None:
 
     # TODO(mc, 2022-11-09): implementation detail; deprecate public method
     def next_tip(
-        self, num_tips: int = 1, starting_tip: Optional[Well] = None
+        self,
+        num_tips: int = 1,
+        starting_tip: Optional[Well] = None,
+        nozzle_map: Optional[NozzleMap] = None,
```

Comment:
Whether this new argument stays a `NozzleMap` or becomes an `InstrumentContext` or whatever, let's make it keyword-only to make future API changes easier.

```suggestion
        self,
        num_tips: int = 1,
        starting_tip: Optional[Well] = None,
        *,
        nozzle_map: Optional[NozzleMap] = None,
```